src/bin/report_collector.rs

```diff
@@ -282,5 +279,23 @@ async fn ipa(
 
     tracing::info!("{m:?}", m = ipa_query_config);
 
-    validate(expected, actual)
+    validate(&expected, &actual);
```

Comment:
Should we log output irrespective of validation?

Reply:
I thought no, we should not because if results are not valid, they shouldn't be recorded

Reply:
In that case, should we log that the results obtained were not as expected? I was thinking this might be useful to debug failures in future

Reply:
I'll probably leave it out of scope for now - lets see if we get those often enough to have a need for it